lib/oli_web/templates/resource/page_preview.html.eex

```diff
@@ -7,10 +7,11 @@
 
 <%= render OliWeb.PageDeliveryView, "_objectives.html", objectives: @objectives %>
 
-<div id="eventIntercept">
+<div id="eventIntercept" class="tex2jax_process">
```

Comment:
Tell mathjax to process this element and initialize the typesetting script (this is required for dynamic content since MathJax runs on page render)